src/mtx3/utils/CommandBinaryBuffer.ts

```diff
@@ -1101,7 +1101,71 @@ export interface IOperatorParameters {
     displaySet4: IDisplaySet4OperatorParameter
 }
 
+export interface IDefine1OperatorParameterExtended {
+    /**
+     * Reset daily maximum power using button.
+     */
+    RESET_DAY_MAX_POWER_KEY: boolean,
+
+    /**
+     * Reset monthly maximum power using button.
+     */
+    RESET_MONTH_MAX_POWER_KEY: boolean,
+
+    /**
+     * `1` - optoport is unlocked by button, `0` - optoport is unlocked.
+     */
+    BLOCK_KEY_OPTOPORT: boolean,
+
+    /**
+     * `1` - constant magnetic field screen
+     *
+     * since build `302.35.005`
+     */
+    MAGNET_SCREEN_CONST: boolean
+
+    /**
+     * Allow display indication in battery mode (`1` - enabled).
+     *
+     * since build `0.0.17`
+     */
+    ALLOW_BROWNOUT_INDICATION: boolean
+}
+
+export interface IOperatorParametersExtended {
+    /**
+     * Timeout for automatic relay activation based on `IMAX`, `PMAX`, `IDIFF`, `COSFI`, minutes.
+     */
+    timeoutRelayOn: types.TUint8,
+
+    /**
+     * Setting for optoport, constant magnetic field screen, resetting power, and battery mode indication.
+     */
+    define1: IDefine1OperatorParameterExtended,
+
+    /**
+     * Timeout for relay activation based on `IMAX`, `PMAX`, `IDIFF`, `COSFI`, seconds.
+     */
+    timeoutRelayKey: types.TUint8,
+
+    /**
+     * Timeout for relay activation upon restoration of quality voltage, seconds.
```

Comment:
```suggestion
     * Timeout for relay activation upon restoration of quality voltage, seconds.
```

voltage or seconds, or "in seconds"?